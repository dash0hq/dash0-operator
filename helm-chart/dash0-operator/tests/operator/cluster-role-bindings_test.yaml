suite: test cluster role bindings
templates:
  - operator/cluster-role-bindings.yaml
tests:
  - it: cluster role bindings should match snapshot
    asserts:
      - matchSnapshot: {}