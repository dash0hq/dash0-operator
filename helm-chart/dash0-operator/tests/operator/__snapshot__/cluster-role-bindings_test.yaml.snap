cluster role bindings should match snapshot:
  1: |
    apiVersion: rbac.authorization.k8s.io/v1
    kind: ClusterRoleBinding
    metadata:
      labels:
        app.kubernetes.io/component: rbac
        app.kubernetes.io/instance: manager-rolebinding
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: clusterrolebinding
        app.kubernetes.io/part-of: dash0-operator
        app.kubernetes.io/version: 0.0.0
        helm.sh/chart: dash0-operator-0.0.0
      name: dash0-operator-manager-rolebinding
    roleRef:
      apiGroup: rbac.authorization.k8s.io
      kind: ClusterRole
      name: dash0-operator-manager-role
    subjects:
      - kind: ServiceAccount
        name: dash0-operator-controller-manager
        namespace: NAMESPACE
  2: |
    apiVersion: rbac.authorization.k8s.io/v1
    kind: ClusterRoleBinding
    metadata:
      labels:
        app.kubernetes.io/component: kube-rbac-proxy
        app.kubernetes.io/instance: proxy-rolebinding
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: clusterrolebinding
        app.kubernetes.io/part-of: dash0-operator
        app.kubernetes.io/version: 0.0.0
        helm.sh/chart: dash0-operator-0.0.0
      name: dash0-operator-proxy-rolebinding
    roleRef:
      apiGroup: rbac.authorization.k8s.io
      kind: ClusterRole
      name: dash0-operator-proxy-role
    subjects:
      - kind: ServiceAccount
        name: dash0-operator-controller-manager
        namespace: NAMESPACE
